src/include/miopen/graphapi/rng.hpp

```diff
@@ -115,7 +115,6 @@ class BackendRngDescriptor : public BackendDescriptor
 private:
     RngBuilder mBuilder;
     Rng mRng;
-
 public:
```

Comment:
```suggestion

public:
```
Triggers clang-format violation. Seems to be changed by an accident. Could you undo this chunk, please?

Reply:
fixed